src/pages/settings/Members.tsx

```diff
@@ -0,0 +1,284 @@
+                {translate('text_63208b630aaf8df6bbfb265d')}
+              </Typography>
+              {!!hasInvites && (
+                <Button
+                  variant="secondary"
+                  onClick={() => {
+                    createInviteDialogRef.current?.openDialog()
+                  }}
+                >
+                  {translate('text_63208b630aaf8df6bbfb265b')}
+                </Button>
+              )}
+            </>
+          )}
+        </Head>
+      )}
+      {!!invitesLoading ? (
+        <LoadingListWrapper>
+          {[1, 2].map((i) => (
+            <InviteItemSkeleton key={i} />
+          ))}
+        </LoadingListWrapper>
+      ) : (
+        <InvitationsListWrapper>
+          {!invitesLoading && !!invitesError ? (
+            <ErrorPlaceholder
+              noMargins
+              title={translate('text_6321a076b94bd1b32494e9e6')}
+              subtitle={translate('text_6321a076b94bd1b32494e9e8')}
+              buttonTitle={translate('text_6321a076b94bd1b32494e9ea')}
+              buttonVariant="primary"
+              buttonAction={invitesRefetch}
+              image={<ErrorImage width="136" height="104" />}
+            />
+          ) : (
+            <>
+              <ListWrapper>
+                {invitesData?.invites.collection.map((invite, i) => (
+                  <InviteItem
+                    key={`invite-item-${i}`}
+                    ref={revokeInviteDialogRef}
+                    invite={invite}
+                  />
+                ))}
+              </ListWrapper>
+              {(invitesMetadata?.currentPage || 0) < (invitesMetadata?.totalPages || 0) && (
+                <Loadmore>
+                  <Button
+                    variant="quaternary"
+                    onClick={() =>
+                      invitesFetchMore({
+                        variables: { page: (invitesMetadata?.currentPage || 0) + 1 },
+                      })
+                    }
+                  >
+                    <Typography variant="body" color="grey600">
+                      {translate('text_63208bfc99e69a28211ec7fd')}
+                    </Typography>
+                  </Button>
+                </Loadmore>
+              )}
+            </>
+          )}
+        </InvitationsListWrapper>
+      )}
+
+      <Head>
+        {!!membersLoading ? (
+          <TitleSkeleton variant="text" height={12} width={160} />
+        ) : (
+          <>
+            <Typography variant="subhead">{translate('text_63208b630aaf8df6bbfb266f')}</Typography>
+            {!hasInvites && (
+              <Button
+                variant="secondary"
+                onClick={() => {
+                  createInviteDialogRef.current?.openDialog()
+                }}
+              >
+                {translate('text_63208b630aaf8df6bbfb265b')}
+              </Button>
+            )}
+          </>
+        )}
+      </Head>
+      {!!membersLoading ? (
+        <LoadingListWrapper>
```

Comment:
On an infinite scroll, we put the loading state at the end of the list so when you "fetchMore", you can still see the items previously fetched

And don't forget to add `notifyOnNetworkStatusChange` on your query so you'll see the loading state change

Reply:
Veery good points, thanks. I updated the code